streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -177,6 +174,9 @@ public class StreamThreadTest {
     @Mock
     private Consumer<byte[], byte[]> mainConsumer;
 
+    @Mock
+    private Consumer<byte[], byte[]> consumer;
```

Comment:
The reason for moving this here as a field is to not have to use @SuppressWarnings in all the tests because of the type <byte[], byte[]> which isn't picked up easily from the mock() function. If this explanation did not make sense feel free to try the change and you will see what I mean :)